### PR TITLE
added `get_staking_leaderboard`

### PR DESCRIPTION
**Pull request by wsteitz**
retrieves the stakes for a given round (defaults to current). Only users participating in the staking competition are returned

### CODE REVIEW

File: tests/test_numerapi.py [hunk 1]
```diff
@@ -43,6 +43,13 @@ def test_get_leaderboard():
     assert len(lb) == 1425
 
 
+def test_get_staking_leaderboard():
+    api = NumerAPI()
+    stakes = api.get_staking_leaderboard(83)
+    # 77 people staked that round
+    assert len(stakes) == 77
```
Comment (philipcmonk) [inline tests/test_numerapi.py]: Round 83 hasn't ended yet, so that number is still going up (78 now).

Comment (wsteitz) [reply tests/test_numerapi.py]: Argh, I thought staking is closed. I'll change that

Comment (wsteitz) [reply tests/test_numerapi.py]: fixed.

**Comment by philipcmonk** [comment]
Looks great!

**Pull request merged**